### PR TITLE
added some nice constructors for selector and sequence

### PR DESCRIPTION
**Pull request by RobotJente**
Constructors for selector and sequence that make it clearer what the children are. Right now the order is set by the way you call "addChild(Node node)" but I think this adds too many lines of code and reduces readability. Therefore I made this constructor, which makes it clear what nodes are in what order. Examle usage would be:
Attack attack = new Attack(bla, blob);
GTP go = new GTP(here, there);
IsARobot isARobot = new IsARobot(this);

Selector select = new Selector(attack, go, isARobot);

### CODE REVIEW

File: include/roboteam_ai/bt/composites/Selector.hpp [hunk 1]
```diff
@@ -6,6 +6,7 @@ namespace bt {
 
 class Selector : public Composite {
 public:
+    Selector(std::vector<std::shared_ptr<bt::Node>> childs);
```
Comment (mrlukasbos) [inline include/roboteam_ai/bt/composites/Selector.hpp]: "children" ;-)

**Review by mrlukasbos — APPROVED** [review]
Easy but smart improvement, especially in context with the task at hand. Good PR!

File: include/roboteam_ai/bt/Node.hpp [hunk 2]
```diff
@@ -39,6 +39,11 @@ class Node {
 
         using Ptr = std::shared_ptr<Node>;
 
+        /**
+         * The update function executes the node. This function is overwritten a lot, most interestingly in the Selector/Sequence/Inverter nodes,
+         * since these nodes can have children. Therefore the update function of these nodes ticks their children.
+         * @return Status result of the update (Running, Success, Failure)
+         */
```
Comment (HaicoDorenbos) [inline include/roboteam_ai/bt/Node.hpp]: :+1:

**Review by HaicoDorenbos — APPROVED** [review]
Looks fine to me

**Pull request merged**